lib/client.js

```diff
@@ -267,7 +272,9 @@ Client.prototype.ready = function() {
  * @returns {Promise}
  */
 Client.prototype.delete = function(key, cb) {
-    assert(key, 'Cannot delete without key!');
+    misc.assert(key, 'Cannot delete without key!');
```

Comment:
Why not roll this into validateKey?

Could give validateKey an optional second argument which is the specific string to respond with. If present it it used in the assert, if not it defaults to something generic. This way it'd be a clean 1 line to validate the key rather than both the misc.assert() and the validateKey()

validateKey(key, 'Cannot delete without key!')